data/presets/golf/cartpath.json

```diff
@@ -12,8 +14,11 @@
     },
     "addTags": {
         "golf": "cartpath",
-        "golf_cart": "designated",
-        "highway": "service"
+        "highway": "path",
+        "golf_cart": "customers"
```

Comment:
Sorry for splitting hairs, but I think `designated` would avoid making assumptions we can’t be sure about. It’s very common for [golf courses](https://www.openstreetmap.org/relation/49218) to be intermingled with residential neighborhoods, in which case the golf course is also open to residents or even the general public, not just members of the country club. There’s even [a town near Atlanta](https://lists.openstreetmap.org/pipermail/tagging/2020-December/057429.html) that maintains a public network of golf cart paths as a major local mode of transportation.

Reply:
All I can say is that golf courses intermingled with residential subdivisions are the norm in the U.S. My only experience with them comes from riding the school bus on a long route that included many of them.

Wouldn’t the more conservative option be `yes`? I’m sure there are country clubs that are strict enough to care that a non-member has brought their golf cart all the way there, but I’d imagine more are preoccupied with making sure golf carts stay on the paths in the first place.

Reply:
`yes` sounds like a good compromise. changed in b41923c2ccdad057e4a5c7550471c466550044ab